src/literal.h

```diff
@@ -128,6 +142,7 @@ class Literal {
   }
   std::array<uint8_t, 16> getv128() const;
   Name getFunc() const { return func; }
+  const ExceptionPackage& getExceptionPackage() const { return *exn.get(); }
```

Comment:
Can we check the `type` is correct in this getter? Seems like we could in the one before it too, so maybe I'm forgetting something...

Reply:
Added assertions in both functions.  I added `getFunc()` when I added support for the reference types and I had forgotten to add an assertion then I think.